pkg/infrastructure/aws/clusterapi/aws.go

```diff
@@ -250,9 +250,11 @@ func removeSSHRule(ctx context.Context, cl k8sClient.Client, infraID string) err
 		if sg, ok := c.Status.Network.SecurityGroups[capa.SecurityGroupControlPlane]; ok {
```

Comment:
Is the security group for bootstrap SSH specified here? https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/api/v1beta1/network_types.go#L165

If not, should it be? If yes, should it be deleted from NetworkSpec?

Reply:
No, the SSH rule is added to the Control Plane Security Group (created by CAPA). We don't want to delete the security group, just the rule.

Reply:
Creating a separate security group for bootstrap is a good idea, but not what we're currently doing.  #8425 will delete the rule without using capi.